src/main/telemetry/frsky.c

```diff
@@ -477,12 +473,6 @@ void initFrSkyExternalTelemetry(frSkyTelemetryInitFrameFn *frSkyTelemetryInitFra
     frSkyTelemetryWrite = frSkyTelemetryWriteExternal;
 }
 
-void deinitFrSkyExternalTelemetry(void)
```

Comment:
Why are you removing this?

Reply:
Its dead code, function without use

Reply:
Yes. But it is the counterpart to `initFrSkyExternalTelemetry`, and once the SPI RX code supports deinitialisation, it will be used.

Reply:
OK to be less confusing i closed it to #if defined(USE_RX_FRSKY_D)